I can't send a PM to the player from the console

Here is the error:
[17:58:36 WARN]: Unexpected exception while parsing console command "pm Lepsi_krut тест"
org.bukkit.command.CommandException: Unhandled exception executing 'pm Lepsi_krut тест' in ru.mrbrikster.chatty.commands.pm.MsgCommand(msg)
at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:169) ~[pufferfish-api-1.18.2-R0.1-SNAPSHOT.jar:?]
at org.bukkit.craftbukkit.v1_18_R2.CraftServer.dispatchCommand(CraftServer.java:906) ~[pufferfish-1.18.2.jar:git-Pufferfish-71]
at org.bukkit.craftbukkit.v1_18_R2.CraftServer.dispatchServerCommand(CraftServer.java:869) ~[pufferfish-1.18.2.jar:git-Pufferfish-71]
at net.minecraft.server.dedicated.DedicatedServer.handleConsoleInputs(DedicatedServer.java:513) ~[pufferfish-1.18.2.jar:git-Pufferfish-71]
at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:487) ~[pufferfish-1.18.2.jar:git-Pufferfish-71]
at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1457) ~[pufferfish-1.18.2.jar:git-Pufferfish-71]
at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1227) ~[pufferfish-1.18.2.jar:git-Pufferfish-71]
at net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:317) ~[pufferfish-1.18.2.jar:git-Pufferfish-71]
at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.ClassCastException: class com.destroystokyo.paper.console.TerminalConsoleCommandSender cannot be cast to class org.bukkit.entity.Player (com.destroystokyo.paper.console.TerminalConsoleCommandSender and org.bukkit.entity.Player are in unnamed module of loader java.net.URLClassLoader @42110406)
at ru.mrbrikster.chatty.commands.pm.PrivateMessageCommand.handlePrivateMessage(PrivateMessageCommand.java:123) ~[Chatty.jar:?]
at ru.mrbrikster.chatty.commands.pm.MsgCommand.handle(MsgCommand.java:62) ~[Chatty.jar:?]
at ru.mrbrikster.chatty.shaded.baseplugin.commands.BukkitCommand.execute(BukkitCommand.java:47) ~[Chatty.jar:?]
at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[pufferfish-api-1.18.2-R0.1-SNAPSHOT.jar:?]
... 8 more

allow-console is true in config.yml

--- a/spigot/src/main/java/ru/mrbrikster/chatty/commands/pm/PrivateMessageCommand.java
+++ b/spigot/src/main/java/ru/mrbrikster/chatty/commands/pm/PrivateMessageCommand.java
@@ -120,7 +120,7 @@ public abstract class PrivateMessageCommand extends BukkitCommand {
                 recipient.sendMessage(recipientFormat);
             } else {
                 FormattedMessage formattedMessage = new FormattedMessage(recipientFormat);
-                formattedMessage.toFancyMessage().send((Player) recipient, (Player) sender);
+                formattedMessage.toFancyMessage().send(recipient, sender instanceof Player ? ((Player) sender) : null);
 
                 String soundName = configuration.getNode("pm.sound").getAsString(null);
                 if (soundName != null) {

